client/src/node.rs

```diff
@@ -291,6 +291,13 @@ impl Node {
         Ok(self.consensus.get_header().clone())
     }
 
+    pub fn get_coinbase(&self) -> Result<CoinbaseAddress> {
```

Comment:
Can we just have it return an ethers-rs `Address`? The consensus type is more just for internal use and has some special serializations. Trying to use ethers types as the outputs when possible as most people are familiar with that. Also means we won't need to export a new type.

Same goes for in `Client` and `Rpc`.

Other than that this all looks good.

Reply:
Ok!

Reply:
Done, please let me know if the implementation is ok